src/containers/my-cooperations/cooperation-notes/note-view/NoteView.tsx

```diff
@@ -97,19 +97,25 @@ const NoteView: FC<NoteViewProps> = ({
       </MenuItem>
     ))
 
+  const userPhoto = note.author.photo
+    ? `${import.meta.env.VITE_APP_IMG_USER_URL}${note.author.photo}`
+    : undefined
+  const isNameValid = note.author.firstName && note.author.lastName
+  const userInitials =
```

Comment:
What do you think about creating global helper for user initials? Because I noticed usage of it here and in one place before. Also it is used in profile icon and i think in the future it can also be reused

Reply:
yes, I created a separate task related to this, because it needs to be changed in many places, not only in profile and notes

Reply:
Ok, i created issue for that